netlify.toml

```
@@ -0,0 +1,10 @@
+[build]
+  base = "web"
+  publish = "dist"
+
+  command = "yarn run build"
+
+    [build.environment]
+    VUE_APP_API_ROOT = "https://girder.dandiarchive.org/api/v1"
+    VUE_APP_PUBLISH_API_ROOT = "https://publish.dandiarchive.org/api/"
+    VUE_APP_SENTRY_DSN = "https://425b9a012300493d867e97785fae7b88@o308436.ingest.sentry.io/5196549"
```

Indentation suggestion

```suggestion
[build]
base = "web"
publish = "dist"
command = "yarn run build"

  [build.environment]
  VUE_APP_API_ROOT = "https://girder.dandiarchive.org/api/v1"
  VUE_APP_PUBLISH_API_ROOT = "https://publish.dandiarchive.org/api/"
  VUE_APP_SENTRY_DSN = "https://425b9a012300493d867e97785fae7b88@o308436.ingest.sentry.io/5196549"
```